cmd/submit.go

```diff
@@ -176,18 +177,14 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	body := &bytes.Buffer{}
 	writer := multipart.NewWriter(body)
 
-	for _, path := range paths {
-		file, err := os.Open(path)
+	for _, doc := range exercise.Documents {
+		file, err := os.Open(doc.Filepath)
```

Comment:
I haven’t verified this yet. But I think we are losing the ability to submit via full path here. 

doc.Filepath is a relative path so if I submit outside of my track calling open here may result in file not exist error. We should add a test for TestSubmitWithFullPath

Reply:
Good catch! I'll add a test.